rmw_fastrtps_cpp/src/rmw_init.cpp

```diff
@@ -77,6 +113,14 @@ rmw_init(const rmw_init_options_t * options, rmw_context_t * context)
     options->implementation_identifier,
     eprosima_fastrtps_identifier,
     return RMW_RET_INCORRECT_RMW_IMPLEMENTATION);
+
+  // Forward fastrtps log messages into rcl logging system.
+  LogConsumer * my_consumer = new RcutilsLogConsumer();
```

Comment:
Create the consumer directly inside the `unique_ptr` if that's the only use it has.